lib/cmd/common/local.js

```diff
@@ -307,21 +242,19 @@ function local(argv, cb) {
     return cb("Error processing args: " + x + "\nUsage: " + local.usage);
   }
   var appEnvironment = {};
-  if (!fs.existsSync(fhcLocalDir)) {
-    if (!argObj.appid) {
-      return cb("Error : 'fhc initlocal <appId>' or 'fhc local <appId>' never called, see the usage for an explanation.\n\nUsage: " + local.usage);
-    }
+  if (!fs.existsSync(fhcLocalDir) && !argObj.appid) {
```

Comment:
Too bad `fs.access` is a newer API, we're stuck with `fs.stat`, I'll replace it with the async version.